llvm/test/CodeGen/AMDGPU/fmamk_fmaak-t16.mir

```diff
@@ -0,0 +1,31 @@
+# RUN: not llc -mtriple=amdgcn -mcpu=gfx1100 -mattr=+real-true16 -run-pass=machineverifier -filetype=null %s 2>&1 | FileCheck %s -check-prefix=GFX11
+# RUN: not llc -mtriple=amdgcn -mcpu=gfx1100 -mattr=+real-true16 -start-before=greedy,0 -stop-after=virtregrewriter,2 -o - %s 2>&1 | FileCheck %s -check-prefix=GFX11
```

Comment:
I'm not sure what the point of the second run line is, the error seems to come from the MIR parser (instead of the machine verifier, for some reason)? 

Test also belongs in test/MachineVerifier or CodeGen/MIR

Reply:
No, it is not a verifier test. The next patch fixes the error this test shows, and only the second run line remains. This is about our current inability to do RA for these instructions.